clarity-ext-scripts/clarity_ext_scripts/covid/review_update_local.py

```diff
@@ -104,7 +88,6 @@ def _validate(self):
             COVID_RESPONSE_POSITIVE,
             COVID_RESPONSE_NEGATIVE,
             FAILED_BY_REVIEW,
-            FAILED_ENTIRE_PLATE_BY_FAILED_EXTERNAL_CONTROL
```

Comment:
I don't understand why `FAILED_ENTIRE_PLATE_BY_FAILED_EXTERNAL_CONTROL` shouldn't that still be a valid value?

Reply:
The reason for this row was a quick and dirty fix in my first implementation. These valid entries are for what the user manually enters in the udf "Reviewer result". FAIL_ENTIRE_PLATE_... on the other hand is always populated automatically by script logic, and is set on the other udf "rt-pcr covid19 result", which in turn is transferred to the reporting step (via sample level udf).

Reply:
Aha! Got it!